src/config/mongoose_config_validator_toml.erl

```diff
@@ -1043,8 +1043,6 @@ validate_non_empty_atom(Value) when is_atom(Value), Value =/= '' -> ok.
 
 validate_non_empty_string(Value) when is_list(Value), Value =/= "" -> ok.
 
-validate_non_empty_list(Value) when is_list(Value), Value =/= [] -> ok.
```

Comment:
It's nothing serious, but I think such commits should be squashed with the previous ones.